[AZINTS] update event hub forwarder installation docs (#29224)
DataDog/documentation: content/en/logs/guide/azure-logging-guide.md

**US3**: Organizations on the Datadog US3 site can simplify Azure log forwarding using the Azure Native integration. This method is recommended and is configured through the [Datadog resource in Azure][5], replacing the Azure Event Hub process. See the [Azure Native Logging Guide][4] for more details.

<div class="alert alert-info">

> Starting April 30, 2025, Azure no longer supports Node.js 18. To ensure compatibility, update using the Deploy to Azure (ARM) template with the same parameters.

Author: since we only support deploying with the arm template, we should only recommend that now.

Reviewer: It doesn't seem like `ARM` (Azure Resource Manager) is the right acronym here. did you mean something else?

> Starting April 30, 2025, Azure no longer supports Node.js 18. To ensure compatibility, update using the Deploy to Azure (ARM) template with the same parameters.

Reviewer: "Azure ARM" would be redundant here

```suggestion
Starting April 30, 2025, Azure no longer supports Node.js 18. To ensure compatibility, update using the Deploy to Azure Resource Manager (ARM) template with the same parameters.
```

Author: I was thinking either Deploy to Azure template, like the button says, or the actual name of the template: `Azure Resource Manager template` would be good. I'll opt for just the latter for simplicity, since combining the two into `Deploy to Azure Resource Manager` feels more confusing potentially.

</div>

## Setup

{{< tabs >}}

> {{% tab "Event Hub" %}}

Author: removes multiple options for event hub and renames for clarity

To get started, click the button below and fill in the form on Azure Portal. The Azure resources required to get activity logs streaming into your Datadog account will be deployed for you. To forward Activity Logs, set the **Send Activity Logs** option to true.

[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FDataDog%2Fdatadog-serverless-functions%2Frefs%2Fheads%2Fmaster%2Fazure%2Feventhub_log_forwarder%2Fparent_template.json)

### Azure platform logs

> When the template deployment finishes, to send Azure platform logs (including resource logs), create diagnostic settings for each of the log sources, with the destination being the eventhub created by the deployment.

Reviewer: ```suggestion
After the template deployment finishes, set up diagnostic settings for each log source to send Azure platform logs (including resource logs) to the Event Hub created during deployment.
```
